### DISCUSSION TITLE
Search Plugin Optimization

### DISCUSSION THREAD

**Discussion by atrolex** [docsifyjs/docsify#1664] [Ideas]
As I understood, search now looks only in the documents which are linked from the sidebars. We have too many documents to link them all in a sidebar. But It is not a problem to link them all from README,md in docs root and all other README documents. I believe it would be nice, if you could create a parameter, which would enable to search for links in README.md too

We have installed your wonderful software here: https://help.atrocore.com/
We have included all the links in the README.md for docs root directory. These are currently not indexed.

Other improvement could be exclude .md filenames from the the search, it would make the search results more beautiful. You could also add a parameter here, in case somebody need it.

**Comment by trusktr** [comment]
I agree, this should be improved. Besides this runtime idea, another idea is generating a JSON file of search results using the docsify command. 

About the results, they could be improved. 👍 The markdown syntax if titles appears in them.

**Comment by masum-mollik-rocketml** [comment]
Any update about this one?
